src/components/dev-hub/modal.js

```diff
@@ -164,6 +171,17 @@ export const Modal = ({
                         headingStyles={headingStyles}
                     />
                     {children}
+                    {showCloseButton && (
+                        <ButtonContainer>
+                            <Button
+                                hasArrow={false}
+                                onClick={deactivateModal}
+                                primary
+                            >
+                                Close
+                            </Button>
+                        </ButtonContainer>
+                    )}
```

Comment:
for context, this is for the Close button that is displayed when form submission is successful:

<img width="416" alt="Screen Shot 2021-07-21 at 2 09 36 PM" src="https://user-images.githubusercontent.com/46455728/126538468-26de7820-3641-437c-aa12-e52fb6baaafa.png">

Reply:
great suggestion! i think this will definitely make the modal more flexible! im thinking of passing in the button with the button container wrapped around it. im currently using container to center the button, but future close buttons might not want that